android/V8Runtime_shared.cpp

```diff
@@ -812,6 +812,54 @@ namespace facebook { namespace v8runtime {
     return handle_scope.Escape(v8BigIntValue->v8BigInt_.Get(v8::Isolate::GetCurrent()));
   }
 
+  jsi::BigInt V8Runtime::createBigIntFromInt64(int64_t val) {
+    return make<jsi::BigInt>(makeBigIntValue(v8::BigInt::New(GetIsolate(), val)));
+  }
+
+  jsi::BigInt V8Runtime::createBigIntFromUint64(uint64_t val) {
+    return make<jsi::BigInt>(makeBigIntValue(v8::BigInt::NewFromUnsigned(GetIsolate(), val)));
+  }
+
+  bool V8Runtime::bigintIsInt64(const jsi::BigInt&) {
+    // V8 doesn't internally track it
+    return true;
+  }
+
+  bool V8Runtime::bigintIsUint64(const jsi::BigInt& val) {
+    bool lossless {true};
+    bigIntRef(val)->Uint64Value(&lossless);
+    return lossless;
+  }
+
+  uint64_t V8Runtime::truncate(const jsi::BigInt& val) {
+    return bigIntRef(val)->Uint64Value(nullptr);
+  }
+
+  jsi::String V8Runtime::bigintToString(const jsi::BigInt&, int) {
+    throw jsi::JSINativeException("V8Runtime::bigintToString is not implemented!");
```

Comment:
>throw jsi::JSINativeException("V8Runtime::bigintToString is not implemented!"); [](http://example.com/codeflow?start=4&length=79)

As v8::BigInt is an v8::Object, we can probably call ObjectProtoToString .. Worth a try !

Reply:
I just copied what the non-android impl was doing

Reply:
I could do it, but I don't know how we would test it... what is the testing process for android?

Reply:
Will leave it for future consideration.